### PR TITLE
Soc notification intranet post in space test it

### PR DESCRIPTION
**Pull request by balkissallem**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiBasicActionMoveMoveLayoutTestIT.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.ELEMENT_MOVE_PAGE_DESTINATION_LABEL;
+import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.ELEMENT_MOVE_PAGE_SELECT_THE_DESTINATION_LABEL;
+import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.ELEMENT_MOVE_SPACESWITCHER;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+
+@Tag("functional")
+@Tag("wiki")
+public class WikiBasicActionMoveMoveLayoutTestIT extends Base {
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiBasicActionMoveMoveLayoutTestIT.java]: why did you add this class ? this PR is for SOCNotificationIntranetPostInSpaceTestIT

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetPostInSpaceTestIT.java [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+		*Expected Outcome:
+			The notification message is :$AVATAR$USER has posted an activity in the $SPACE space.$ACTIVITY$DATEWhere :
+			- $AVATAR is the thumbnail of User B
+			- $USER is User B
+			- $SPACE is Space 1
+			- $ACTIVITY is the activity title / message
+			- $DATE is the date of the notification*/
+    @Test
+    public void test01_CheckPostOnMySpaceNotification() {
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetPostInSpaceTestIT.java]: this test fails

File: social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/NotificationsAdminSeting.java [hunk 3]
```diff
@@ -421,13 +421,13 @@ public void enableNotification(notificationType notifToDisable) {
...
+      $(ELEMENT_SPACE_NOTIFICATION_POST_EDIT_BTN).click();
       if (evt.waitForAndGetElement(ELEMENT_SPACE_POST_INTRANET_NOTIFICATION_CHECKBOX_CHECKED, 2000, 0) == null)
-        evt.check(ELEMENT_SPACE_POST_INTRANET_NOTIFICATION_CHECKBOX, 2);
+        $(ELEMENT_SPACE_POST_INTRANET_NOTIFICATION_CHECKBOX).click();
       info("Click on Save button");
-      evt.click(ELEMENT_SPACE_POST_NOTIFICATION_SAVE_BTN);
+      $(ELEMENT_SPACE_POST_NOTIFICATION_SAVE_BTN).click();
       info("Verify that Intranet notification is hidded");
-      evt.waitForAndGetElement(ELEMENT_SPACE_POST_INTRANET_NOTIFICATION_TITLE, 3000, 1);
+      $(ELEMENT_SPACE_POST_INTRANET_NOTIFICATION_TITLE).waitUntil(Condition.disappears,Configuration.timeout);
```
Comment (ilyeskallala) [inline social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/NotificationsAdminSeting.java]: must be $(ELEMENT_SPACE_POST_INTRANET_NOTIFICATION_TITLE).waitUntil(Condition.visible,Configuration.timeout);

File: wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/WikiHomePage.java [hunk 4]
```diff
@@ -7,6 +7,7 @@
 import static com.codeborne.selenide.Selenide.executeJavaScript;
 import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.*;
 import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.hamcrest.core.Is.is;
```
Comment (ilyeskallala) [inline wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/WikiHomePage.java]: useless import

**Review by ilyeskallala — CHANGES_REQUESTED** [review]

File: wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/WikiHomePage.java (hunk 4 above)
Comment (balkissallem) [reply wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/WikiHomePage.java]: corrected

File: social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/NotificationsAdminSeting.java (hunk 3 above)
Comment (balkissallem) [reply social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/NotificationsAdminSeting.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetPostInSpaceTestIT.java (hunk 2 above)
Comment (balkissallem) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetPostInSpaceTestIT.java]: done
